python/tests/test_file_echo_api.py

```diff
@@ -178,7 +204,7 @@ class RemoteSocketProcessTests(GenericFileEchoTests, unittest.TestCase):
     @classmethod
     def setUpClass(self):
         p = subprocess.Popen(
-            ["cabal", "run", "exe:file-echo-api", "--verbose=0", "--", "socket", "--port", "50005"],
+            ["cabal", "run", "exe:file-echo-api", "--verbose=0", "--", "socket", "--port", "50005", "--log", "remote-socket-server.log"],
```

Comment:
Perhaps use `get_server_log_file()` here instead of duplicating the log file name. (Similarly for other test cases as well.)

Reply:
done